cmd/csaf_aggregator/config.go

```diff
@@ -32,10 +32,7 @@ const (
 	defaultWeb            = "/var/www/html"
 	defaultDomain         = "https://example.com"
 	defaultUpdateInterval = "on best effort"
-)
-
-var (
-	defaultLockFile = "/var/csaf_aggregator/run.lock"
+	defaultLockFile       = "/var/csaf_aggregator/run.lock"
```

Comment:
I'm curious: Is there a specific reason why we choose `/var/csaf_aggregator/run.lock` for the lock file and not `/var/lock/<something>`?